scripts/task/ExportDeliveryResults.php

```diff
@@ -71,6 +75,11 @@ class ExportDeliveryResults implements Action, ServiceLocatorAwareInterface, Wor
     private $submittedVersion;
     private $destination;
     private $filters = [];
+    private $format;
+    /**
+     * @var DeliveryResultsExporterFactoryInterface
+     */
+    private $singleDeliveryResultsExporterFactory;
```

Comment:
maybe just deliveryResultsExporterFactory ? I just can't understand why there should be prefix 'single'